UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Services/BuilderService.cs

```diff
@@ -345,5 +356,49 @@ private async Task<bool> DownloadReadMeFileAsync(string batchId, string exchange
 
             return isDownloadReadMeFileSuccess;
         }
+
+        private async Task UpdateSerialFileAsync(string serialFilePath, string exchangeSetType, string correlationId)
+        {
+            string serialFileContent = String.Empty;
+            const string searchText = "UPDATE";
+            try
+            {
+                serialFileContent = fileSystemHelper.ReadFileText(serialFilePath);
```

Comment:
can declared closer to usage
```suggestion
            const string searchText = "UPDATE";
            try
            {
                string serialFileContent = fileSystemHelper.ReadFileText(serialFilePath);
```

Reply:
Resolved